### PR TITLE
Wiki plugin

### LINKED ISSUE #2551: Wiki lookup
The idea of when you Examine a monster it pulls the information of what it drops and displays in a table from OSRS Wiki, found this on google aware its a private server but this kind of idea.

https://gyazo.com/97dcc89fa21f349e586569a2ffb8f1e7

### PR DESCRIPTION
**Pull request by Abextm**
cast_wiki
quests
search

Supersedes/Closes #2829
Closes #2551

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java [hunk 1]
```diff
@@ -426,8 +427,17 @@ public static SpritePixels getImageSpritePixels(BufferedImage image, Client clie
 
 		try
 		{
-			new PixelGrabber(image, 0, 0, image.getWidth(), image.getHeight(), pixels, 0, image.getWidth())
-				.grabPixels();
+			PixelGrabber g = new PixelGrabber(image, 0, 0, image.getWidth(), image.getHeight(), pixels, 0, image.getWidth());
+			g.setColorModel(new DirectColorModel(32, 0xff0000, 0xff00, 0xff, 0xff000000));
+			g.grabPixels();
+
+			for (int i = 0; i < pixels.length; i++)
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java]: What's this doing? Why is it in this commit?

File: runelite-client/src/main/java/net/runelite/client/game/chatbox/ChatboxTextInput.java [hunk 2]
```diff
@@ -95,12 +95,13 @@ private static IntPredicate getDefaultCharValidator()
 	@Getter
 	private int fontID = FontID.QUILL_8;
 
+	@Getter
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/game/chatbox/ChatboxTextInput.java]: The commit description should contain a reason why this is being changed

File: runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java (hunk 1 above)
Comment (Abextm) [reply runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java]: It ensures any fully transparent pixels are full black, so the sprite draw routines draw them as transparent

File: runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiSprite.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import net.runelite.client.game.SpriteOverride;
+
+@RequiredArgsConstructor
+public enum WikiSprite implements SpriteOverride
+{
+	WIKI_ICON(-300, "wiki.png"),
+	WIKI_SELECTED_ICON(-301, "wiki_selected.png"),
+	FIXED_MODE_MINIMAP_CLICKMASK(1200, "fixed_mode_minimap_clickmask.png");
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiSprite.java]: Should this be a `SpriteID`?

**Pull request merged**